test/common/mocks/event/mocks.cc

```diff
@@ -16,8 +16,5 @@ using testing::SaveArg;
 namespace Envoy {
```

Comment:
is this file necessary?

Reply:
No, but will be adding to it shortly when I add unit tests, so it's a placeholder.